tests/test_recreation.py

```diff
@@ -927,8 +925,8 @@ def test_existing_output_shapefiles(self):
         expected_grid_vector_path = os.path.join(
             REGRESSION_DATA, 'hex_grid_vector_path.shp')
 
-        pygeoprocessing.testing.assert_vectors_equal(
-            out_grid_vector_path, expected_grid_vector_path, 1E-6)
+        _assert_vector_attributes_eq(

```

Comment:
This test, and some of the previous that I didn't comment on, I think are okay with only asserting on the attribute values since geometry creation is covered by the other tests I commented on.